packages/sui-widget-embedder/file-templates/_index.scss.js

```diff
@@ -0,0 +1 @@
+module.exports = (sitePrefix) => `@import '~@schibstedspain/${sitePrefix}-theme/lib/index.scss';`
```

Comment:
hmmmm an open-source tool that creates references to `@schibstedspain`.
@carlosvillu

Reply:
We are doing the same here
https://github.com/SUI-Components/sui/blob/master/packages/sui-studio/bin/sui-studio-generate.js#L144

We have to define a policy for this kind of thing. Meanwhile I would not change it.

But It is true, that can`t be in an open source repo.

Reply:
We are doing the same here
https://github.com/SUI-Components/sui/blob/master/packages/sui-studio/bin/sui-studio-generate.js#L144

We have to define a policy for this kind of thing. Meanwhile I would not change it.

But It is true, that can`t be at an open source repo.